packages/components/src/sandbox/index.js

```diff
@@ -232,7 +233,7 @@ class Sandbox extends Component {
 				title={ title }
 				className="components-sandbox"
 				sandbox="allow-scripts allow-same-origin allow-presentation"
-				onLoad={ this.trySandbox }
+				onLoad={ () => this.trySandbox() }
```

Comment:
Could you test this to make sure it's not causing unnecessary re-renders down the tree? Since this component is already a class, then to avoid re-renders we might as well add:

```js
constructor() {
  …
  this.onLoad = this.trySandbox.bind( this, false );
}

…
<FocusableIframe
  onLoad={ this.onLoad }
```

Reply:
@mcsf I just tested! Both solutions (the one I pushed) and the one you proposed, have the same amount of renders for both `Sandbox` and `FocusableIframe` (4 for me).

As far as implementation, I'm happy to go with either 👍 .

Just lemme know :)

Reply:
Thanks for checking — merge away :)

Reply:
Oh, actually, make it an explicit `=> this.trySandbox( false )`!